odxtools/cli/dummy_sub_parser.py

```diff
@@ -14,19 +15,19 @@ class DummyTool:
     should bail out.
     """
 
-    def __init__(self, tool_name, error):
+    def __init__(self, tool_name: str, error: Exception):
         self._odxtools_tool_name_ = tool_name
         self._error = error
 
-    def add_subparser(self, subparser_list):
+    def add_subparser(self, subparser_list: argparse._SubParsersAction) -> None:
```

Comment:
`_SubParsersAction` is private

Reply:
yes, but how to annotate this correctly? `Any`?

Reply:
## Option 1
https://github.com/python/typeshed/issues/7539#issuecomment-1076581049

## Option 2

Even if python makes changes and `_SubParsersAction` is no longer there, only mypy will report it, but the code itself will still work.

```suggestion
    def add_subparser(self, subparser_list: "argparse._SubParsersAction") -> None:
```

Reply:
cool. I picked option two: e8009c70294dc4c2b10c95a9011da0e3e6fe9d48